portfolio/pom.xml

```diff
@@ -0,0 +1,62 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Should this directory be named `portfolio`?

Reply:
No, that's just the project root. It has to be there in all pom files according to the documentation. It was like that in the training project as well.